fenix/app/src/test/java/org/mozilla/fenix/components/metrics/DefaultMetricsStorageTest.kt

```diff
@@ -382,6 +384,120 @@ class DefaultMetricsStorageTest {
         assertTrue(updateSlot.captured > 0)
     }
 
+    @Test
+    fun `GIVEN first week activated threshold reached THEN will be sent`() = runTest(dispatcher) {
```

Comment:
It'll be pretty lengthy, but I think we should update this test name to make it clear that both conditions (days of use & search) are required.

Reply:
updated the title to reflect both conditions are met.